index.html

```diff
@@ -12,46 +12,52 @@
   <link rel="stylesheet" href="style.css" >
 </head>
 <body>
+<a href="#"><img id="logo" src="https://img2.freepng.ru/20180423/wlw/kisspng-soldier-computer-icons-military-salute-5ade61d71d3dc2.8402920815245234791198.jpg"></a>
 <h1>Hello fellow!</h1>
 <h2>This is a page where you are able to learn current information about mobilization in Ukraine!</h2>
+<form class="was-validated">
 <div class="input-group mb-3" id="first-div">
```

Comment:
is not so good idea to name like "first-div", "second-div" ect it make no sence.. If u realy need that pls rename to some nane that make sence, for instance for first-div is coul be "gender-container"

Reply:
yeah, you're right